packages/components/modal/src/ModalConfirm/ModalConfirm.tsx

```diff
@@ -174,28 +191,19 @@ export function ModalConfirm({
       shouldCloseOnOverlayClick={shouldCloseOnOverlayClick}
       shouldCloseOnEscapePress={shouldCloseOnEscapePress}
       allowHeightOverflow={allowHeightOverflow}
+      initialFocusRef={cancelRef}
     >
       {() => {
         return (
-          <div>
+          <>
             <Modal.Header title={title || ''} {...modalHeaderProps} />
             <Modal.Content {...modalContentProps}>{children}</Modal.Content>
             <Modal.Controls {...modalControlsProps}>
-              {modalControlsProps?.position === 'right' ? (
-                <Fragment>
-                  {cancelButton}
-                  {secondaryButton}
-                  {confirmButton}
-                </Fragment>
-              ) : (
-                <Fragment>
-                  {confirmButton}
-                  {secondaryButton}
-                  {cancelButton}
-                </Fragment>
-              )}
+              {cancelButton}
+              {secondaryButton}
+              {confirmButton}
```

Comment:
Will we still support three buttons in the modal?

Reply:
Yeah, a good question. I'd remove `secondaryButton` completely and make "cancelButton" to look like a secondary

Reply:
Removed secondary button after talking to @fabe and @domarku